src/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -351,8 +353,7 @@ private bool AcquireClientCredentials(ref byte[] thumbPrint)
                 X509Certificate2 remoteCert = null;
                 try
                 {
-                    X509Certificate2Collection dummyCollection;
-                    remoteCert = CertificateValidationPal.GetRemoteCertificate(_securityContext, out dummyCollection);
+                    remoteCert = CertificateValidationPal.GetRemoteCertificate(_securityContext, out X509Certificate2Collection dummyCollection);
```

Comment:
Where is `dummyCollection` used?  It seems like it's not, in which case it could use a discard.  However, do we have an existing issue here around cleaning up anything in this collection?  @bartonjs, should we be disposing of the contents of the collection?

Reply:
Yeah, if it's non-null then everything in it should be Disposed.  The only other caller that I see of GetRemoteCertificate already disposes it (this one is probably a lot less used).

Reply:
Ok.  We can fix it separately.